src/main/external-resources/renderers/Google-ChromecastUltra.conf

```diff
@@ -41,12 +41,5 @@ Supported = f:m4a            n:2      a:aac-lc|he-aac|mp3   m:audio/mp4
 Supported = f:wav            n:2      a:lpcm                m:audio/x-wav
 Supported = f:weba           n:2      a:opus|vorbis         m:audio/webm
 
-# Supported image formats:
```

Comment:
It's not used anymore since #1061. #1061 removed it from all the other renderers, but when we merge old PR's they are still around.